4-projects/modules/ml_env/example_vertex.tf

```
@@ -28,6 +28,8 @@ locals {
     "cloudbuild.googleapis.com",
     "notebooks.googleapis.com"
   ]
+  ml_pipeline_sa     = try(local.app_infra_pipeline_service_accounts["ml-machine-learning"], null)
+  enable_ml_bindings = try(local.ml_pipeline_sa != null && local.ml_pipeline_sa != "", false)
```

```suggestion
  enable_ml_bindings = local.ml_pipeline_sa != null && local.ml_pipeline_sa != ""
```
It does not need a try()